app/assets/javascripts/discourse/app/mixins/open-composer.js

```diff
@@ -39,14 +39,15 @@ export default Mixin.create({
     });
   },
 
-  openComposerWithMessageParams(recipients, topicTitle, topicBody) {
+  openComposerWithMessageParams(recipients, topicTitle, topicBody, hasGroups) {
```

Comment:
Can we convert these arguments to an object so it's more readable? See one of the call sites:

```javascript
this.send(
      "createNewMessageViaParams",
      this.get("model.name"),
      "",
      "",
      true
    );
```

I have no idea what those parameters are! It would be so much nicer like:

```javascript
  this.send(
    "createNewMessageViaParams",
    { recipients: this.get("model.name"),
       hasGroups: true
    });
```

(assuming we omit `""` params).